pkgs/top-level/python-packages.nix

```diff
@@ -6038,6 +6038,29 @@ let
     };
   };
 
+  django_redis = makeOverridable ({ django ? self.django }: buildPythonPackage rec {
```

Comment:
But every other `django-something` package in nixpkgs is named `django_something` so that's what I was following.